far/editor.cpp

```diff
@@ -3806,6 +3806,8 @@ bool Editor::Search(bool Next)
 
 				case KEY_CTRLENTER:
 				case KEY_RCTRLENTER:
+				case KEY_CTRL|KEY_MSLCLICK:
+				case KEY_RCTRL|KEY_MSLCLICK:
```

Comment:
Would it always work as expected - say, if you click the border or outside the menu?

Reply:
Good question. Somewhat surprisingly, the behavior is reasonable.

Gesture | Baseline | My Change
--- | --- | ---
Ctrl+Click outside | Closes dialog | Closes dialog
Ctrl+Click on border frame | Does nothing | Jumps to the currently selected item without changing the item *)
Ctrl+Click on border margin (outside of the frame) | Moves dialog | Jumps to the currently selected item without changing the item *)
Ctrl+Drag from border outside | Closes dialog | Closes dialog

*) Admittedly, this behavior is somewhat unexpected. Will it do, or should I try to filter out Ctrl+Click on the border?

Reply:
Thanks for checking.
I'm asking because mouse events are usually handled via ProcessMouse(), using KEY_MSLCLICK is quite rare.
Perhaps it's ok in this case.

Reply:
Thank you for the explanations. I'll think about it. I do not quite like the behavior on the border.

Reply:
Let's proceed with KEY_MSLCLICK, it's good enough for now.